src/main/java/com/snowflake/kafka/connector/internal/telemetry/SnowflakeTelemetryService.java

```diff
@@ -77,8 +79,13 @@ public void reportKafkaConnectStart(
       final long startTime, final Map<String, String> userProvidedConfig) {
     ObjectNode dataObjectNode = getObjectNode();
 
+    String jdkVersion = System.getProperty("java.version");
+    String jdkDistribution = System.getProperty("java.vendor");
+
     dataObjectNode.put(START_TIME, startTime);
     dataObjectNode.put(KAFKA_VERSION, AppInfoParser.getVersion());
+    dataObjectNode.put(JDK_VERSION, jdkVersion);
```

Comment:
Please add some basic assertion to `SnowflakeTelemetryServiceTest`. Not for a specific JDK version of course, but to check if it's a non-empty number or string (or whatever it returns).

Reply:
Added a test